code/modules/overmap/ships/engines/engine.dm

```diff
@@ -17,13 +17,22 @@ var/list/ship_engines = list()
 /datum/ship_engine/proc/burn()
 	return 0
 
+/datum/ship_engine/proc/get_exhaust_velocity()
```

Comment:
are these even used?
these datums are sorta prehistorical extensions, providing interface for external things to get important info / give important orders to various engines (since idea is that not only atmospheric machines can be them)
These nitty gritty things are probably not needed and are better left to engine machines themselves, doesn't seem like anything outside the engine is using them.

Reply:
Yes they are used.

Reply:
where?
I only see get_exhaust_velocity() called, rest are used inside the engine machine

Reply:
In code that is not pushed. I literally added get_exhaust_velocity() as I need it for stuff down the line.